cumulus_library/studies/core/core_templates/patient.sql.jinja

```diff
@@ -34,24 +34,32 @@ WITH temp_patient AS (
         patient AS p
     LEFT JOIN core__patient_ext_race AS er ON p.id = er.id
     LEFT JOIN core__patient_ext_ethnicity AS ee ON p.id = ee.id
+),
+
+-- Grab a single address to use for this patient, if present
+temp_address AS (
+    SELECT
+        tp.id,
+        {# FIXME: use 'rank' to prefer an address #}
+        {# complications: no postalcode on best one or no ranks specified #}
+        substr(arbitrary(t_address.addr_row.postalcode), 1, 3) AS postalcode3
```

Comment:
That felt more complicated than I wanted to chew off right now. But this change at least gets us out of duplicate patient rows. Which I assume is worse than an arbitrary address being used? But open to push back there, that this should be fixed differently.

Reply:
in the cases where you were seeing this, is `use` present and populated? we could default to `home`? and then walk down?

regardless, unless someone is commuting by plane, on average i would think that these would be pretty close (especially on the reduction to 3 digits, unless someone is commuting from new hampshire to boston, so this is certainly better than duplicates.

Reply:
Hmm, and even if we do keep `arbitrary()`, we should probably sort first.

Reply:
Oh yeah, all my talk of `rank` was me looking at the wrong struct 🤦 (ContactPoint) - yeah, Address looks like `use` would be what you'd want to cue off, if available. But yeah, I think this is a good first step, and we can optimize in future.